integrations/Rockerbox/browser.js

```diff
@@ -104,6 +104,8 @@ class Rockerbox {
   page(rudderElement) {
     logger.debug("=== In Rockerbox Page ===");
     const { message } = rudderElement;
+    const { anonymousId } = message;
+    message.properties.anonymousId = anonymousId;
     window.RB.track("view", message.properties);
```

Comment:
It's not a good idea to mutate incoming data. Repeat this for the `track` API as well.
```suggestion
    const { anonymousId } = message;
    window.RB.track("view", { ...message.properties, anonymousId });
```

Reply:
Addressed.